engine/src/flutter/lib/web_ui/skwasm/text/paragraph_style.cpp

```diff
@@ -38,7 +38,7 @@ SKWASM_EXPORT void paragraphStyle_setTextDirection(
 }
 
 SKWASM_EXPORT void paragraphStyle_setMaxLines(ParagraphStyle* style,
-                                              size_t maxLines) {
+                                              SkScalar maxLines) {
   style->skiaParagraphStyle.setMaxLines(maxLines);
```

Comment:
It seems that `setMaxLines` is a `size_t` ([link](https://skia.googlesource.com/skia/+/52130e5c3b34/modules/skparagraph/include/ParagraphStyle.h#103)). This call probably truncates the fraction parts. Should we to a `ceil` here instead?